### PR TITLE
Move On Shake up

### PR DESCRIPTION
**Pull request by livcheerful**
fixes https://github.com/microsoft/pxt-microbit/issues/3616

lemme know what you guys think! I can move it to other places if you think that'd be better This is what it looks like [updated]:
image

### CODE REVIEW

**Comment by pelikhan** [comment]
Try using radio and compass in the same program.

________________________________

**Comment by pelikhan** [comment]
I think you want it under the 128, above the "b" label. SHAKE text would be smaller.

File: sim/visuals/microbit.ts [hunk 1]
```diff
@@ -475,7 +475,7 @@ path.sim-board {
                     this.board.accelerometerState.shake();
                 });
                 accessibility.setAria(this.shakeButton, "button", "Shake the board");
-                this.shakeText = svg.child(this.g, "text", { x: 400, y: 110, class: "sim-text" }) as SVGTextElement;
+                this.shakeText = svg.child(this.g, "text", { x: 400, y: 50, class: "sim-text" }) as SVGTextElement;
                 this.shakeText.textContent = "SHAKE"
```
Comment (pelikhan) [inline sim/visuals/microbit.ts:479]: now that we support translations in the sim, we shoud consider translating this or using an icon instead.

**Review by pelikhan — CHANGES_REQUESTED** [review]
Overlaps with radio antenna + RSSI text.

File: sim/visuals/microbit.ts (hunk 1 above)
Comment (livcheerful) [reply sim/visuals/microbit.ts]: I added pxsim.localization.lf() in two places. Is that enough for translation? (or is there a good way for me to test it?)

**Comment by pelikhan** [comment]
Try using the editor in a different locale.

________________________________

**Comment by pelikhan** [comment]
Best to do this in another PR.

________________________________

File: sim/visuals/microbit.ts [hunk 2]
```diff
@@ -72,6 +72,13 @@ namespace pxsim.visuals {
         pointer-events: none;
         }
 
+        .sim-text-small {
+            font-family:"Lucida Console", Monaco, monospace;
+            font-size:20px;
+            fill:#fff;
+            pointer-events: none;
+        }
+
```
Comment (jwunderl) [inline sim/visuals/microbit.ts:81]:
```suggestion
        .sim-text-small,
```

nit: seems like you can combine these two?

**Review by jwunderl — APPROVED** [review]

**Review by pelikhan — APPROVED** [review]
LGTM. Make sure we don't do anything with sim-text-pin in high contrast mode (where we have different sim colors).

File: sim/visuals/microbit.ts (hunk 2 above)
Comment (livcheerful) [reply sim/visuals/microbit.ts]: Joey that is genius

**Comment by livcheerful** [comment]
> LGTM. Make sure we don't do anything with sim-text-pin in high contrast mode (where we have different sim colors).

oh good point! I'll try it out

**Comment by livcheerful** [comment]
High contrast mode doesn't change the simulator at all for me right now. I'm going to complete this PR, see what's wrong with HC, and if it affects the SHAKE text I'll open up a new PR to fix

**Pull request merged**